docs/track.rst

```diff
@@ -122,6 +123,10 @@ In the ``operations`` section you describe which operations are available for th
 
 In the ``schedule`` section you describe the workload for the benchmark, for example index with two clients at maximum throughput while searching with another two clients with ten operations per second. The schedule either uses the operations defined in the ``operations`` block or defines the operations to execute inline.
 
+In the ``challenges`` section you describe more than one set of operations, in the event your track needs to test more than one set of scenarios. This section is optional, and more information on ``challenges`` :ref:`can be found here <track_challenge>`.
```

Comment:
This is a matter of personal preference, I am not a huge fan of the `can be found here` expression in official docs; I'd have said:

```
and more information can be found in the :ref:`challenges section<track_challenge>`. 
```